web/e2e/specs/flows/steps/ingestion.ts

```diff
@@ -247,6 +269,42 @@ export default function (qaProjectDir) {
       await manageFlowPage.removeFlow(flowData.flow5);
     });
 
+    it('Should verify ingest target format', async function () {
+      await manageFlowPage.createFlow(flowData.flow1);
+      await editFlowPage.addStep(flowData.flow1, json);
+      await browser.sleep(5000);
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"json\"");
+      await ingestStepPage.setTargetFileType("XML");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"xml\"");
+      await ingestStepPage.setTargetFileType("Text");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"text\"");
+      await ingestStepPage.setTargetFileType("Binary");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-document_type \"binary\"");
+    });
+
+    it('Should verify target permissions', async function () {
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-output_permissions \"rest-reader,read,rest-writer,update\"");
+      await ingestStepPage.targetPermissions.clear();
+      await ingestStepPage.targetPermissions.sendKeys("rest-reader,read,rest-writer,update,user1,execute");
+      await ingestStepPage.targetPermissions.sendKeys(protractor.Key.ENTER);
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-output_permissions \"rest-reader,read,rest-writer,update,user1,execute\"");
+    });
+
+    it('Should target uri replacement', async function () {
+      await ingestStepPage.targetUriReplace.clear();
```

Comment:
verify missing in the test description and in line 300.
Can you also please add tests that validate the error message for the URI preview when the target uri is wrong. There are 3 error messages

Reply:
Thanks, added missing verify. Protractor can't extract the text value of these fields. I tried to get it using JavaScript but it didn't work also.  I will discuss it with the developer and add tests in the future.